src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
Either char* or "const std::string&" (that should work right?), or even std::string* (a little trickier).
Never an object on the stack. We need to stop doing that. Extremely stupid to copy a whole object instead of just passing the pointer to it. 

Now,

    std::string s = getDbPrefix();

works just fine.
Just like

    std::string s = "27";

What does NOT work is this:

    std::string s = getDbPrefix()  + "abc";

You can't add a char* to a char*.
This only works if you have a std::string to the left.
Then, how much time do we lose on calling a method (operator+) on the std::string every time, instead of a strcat ... that would be nice to know.
Here, the C++ way is SO MUCH nicer to read and that is why we're using it. We lose time though. Probably a lot.